variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+  default = "JP"
+}
+
+variable "docker_url" {
```

Comment:
Makes sense to not be too thorough here, but if this was a real project then adding a sensible description here would be good - even if it feels really obvious to you then `docker_url` could definitely confuse someone; is it the image URL, the repository URL, or something else?

Reply:
Thanks Jack - makes sense - I've renamed it to container_repository_url.